pkg/abcupdater/version.go

```diff
@@ -252,3 +273,39 @@ func updateVersionOutput(updateDetails *versionUpdateDetails) (string, error) {
 
 	return b.String(), nil
 }
+
+func (c *CheckVersionParams) loadLocalCachedData() (*LocalVersionData, error) {
```

Comment:
It feels weird to have this functionality be a method on a "Params" struct. Typically I'd expect a Params struct to have no methods other than getters and setters. Two possibilitiels:

 - Maybe just rename "CheckVersionParams" to "VersionChecker" or "VersionLoader" or something?
 - Make this a top-level function that accepts params as an argument, just like `CheckAppVersion`

Same with setLocalCachedData below.

Reply:
Yeah, that's the java programmer in me, I'll change